app/src/main/res/layout/long_horizontal_item.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
try to use constraint layout, it's more fast in creating

Reply:
fixed.